locale/en_US/manager.po

```diff
@@ -808,11 +808,28 @@ msgstr "Items with DOIs"
 msgid "doi.manager.settings.enableFor"
 msgstr "{$objects} will be assigned a DOI"
 
+msgid "doi.manager.settings.doiPrefix.required"
+msgstr "A DOI prefix is required"
+
+
 msgid "doi.manager.settings.doiSuffix"
 msgstr "DOI Suffix"
 
+msgid "doi.manager.settings.doiSuffix.default.description"
+msgstr "Use the default DOI suffix generation?"
```

Comment:
> DOIs can be generated automatically so that editors do not need to manually create or edit DOIs.

And then instead of the Yes/No answers, use:

> - [ ] Yes, generate unique DOIs automatically.
> - [ ] No, use a custom DOI suffix.